src/popup/components/FungibleTokens/TokensListItem.vue

```diff
@@ -5,7 +5,7 @@
     class="tokens-list-item"
     :class="{ extend: preventNavigation, 'asset-selector': assetSelector }"
     :to="preventNavigation ? null : {
-      name: 'token-details',
+      name: tokenData.contractId === AETERNITY_CONTRACT_ID ? 'coin-details' : 'token-details',
```

Comment:
Now in my opinion there's too much logic here. I suggest to move it to computed proprty.